qiskit_experiments/library/hamiltonian/__init__.py

```diff
@@ -0,0 +1,64 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+===============================================================================================
+Hamiltonian Characterization Experiments (:mod:`qiskit_experiments.library.hamiltonian`)
```

Comment:
Probably `entangler` and `Entangler characterization`? But this could kick out JAZZ experiment from here. I think having static ZZ in the module improves accessibility.